utils/defaults/defaults.go

```diff
@@ -31,6 +31,7 @@ const (
 )
 
 const (
+	DefaultAmbassadorAPIGroup     = "getambassador.io"
 	DefaultAmbassadorVersion      = "v2"
```

Comment:
So it seems this change now accepts API version with or without the group in it.  e.g.:

```
rollouts-controller --ambassador-api-version v2
rollouts-controller --ambassador-api-version getambassador.io/v2
rollouts-controller --ambassador-api-version x.getambassador.io/v3alpha1
```

Now that we support supplying a different group, can we change `DefaultAmbassadorVersion` to be `getambassador.io/v2` so that when users see the `--help` of the rollout controller, they know that this is possible?

Reply:
makes sense.